Main.java

```diff
@@ -1,7 +1,5 @@
-/**
- *
- * @author 
- */
+import java.util.Scanner;
+ 
 public class Main {

```

Comment:
Don't forget to put a description of the program and your name at the top of the program inside of the code header. It looks like you have removed it entirely!